Tests/SparkConnectTests/Resources/queries/describe_database.sql.answer

```diff
@@ -0,0 +1,5 @@
+[Catalog Name,spark_catalog]
+[Namespace Name,default]
+[Comment,default database]
+[Location,*]
+[Owner,*]
```

Comment:
Hmm, why the the second elements of the two items ["Location","file:\/opt\/spark\/work-dir\/spark-warehouse"],["Owner","185"] are `*` now?

Reply:
During the `Catalog` PR, the cleaning methods were added previously. While regenerating, I used the cleaned strings because the cleaned one is better in the GitHub repo.

https://github.com/apache/spark-connect-swift/blob/21724f577a686667779677aa7b0dfbf7a7fd6ca1/Tests/SparkConnectTests/SQLTests.swift#L36-L50

Reply:
I see. Thank you.